client/components/EditPhotoForm/EditPhotoForm.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import axios from 'axios';
+
+class EditPhotoForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      url: ''
+    }
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      url: event.target.value
+    });
+  }
+
+  handleSubmit(event) {
+    let id = window.location.pathname;
+    let endpoint = `/photos${id}`
```

Comment:
on localhost:3003/ this id is null and points to `/photos`.. best to make a failsafe like check if id is blank or null and if so, set id to `1`.